vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -221,6 +271,9 @@ class CreateSpaceViewModel @AssistedInject constructor(
         }
         viewModelScope.launch(Dispatchers.IO) {
             try {
+                val alias = if (state.spaceType == SpaceType.Public) {
+                    if (state.aliasManuallyModified) state.aliasLocalPart else getAliasFromName(state.name)
```

Comment:
`aliasLocalPart` should always be up to date, whatever the value of `aliasManuallyModified` is I think. Why do you call again `getAliasFromName()`?

Reply:
cleaned the code